docs/maintain-sync.md

```diff
@@ -28,25 +28,29 @@ was, or which extrinsics resulted in a certain state change are fast operations
 archive node. However, an archive node takes up a lot of disk space - around Kusama's 1.6 millionth
 block this was around 15 to 20GB.
 
-A **full node** is _pruned_, meaning it discards all information older than 256 blocks, but keeps
-the extrinsics for all past blocks, and the genesis block. A node that is pruned this way requires
-much less space than an archive node. In order to query past state through a full node, a user would
-have to wait for the node to rebuild the chain up until that block. A full node _can_ rebuild the
-entire chain with no additional input from other nodes and become an archive node. One caveat is
-that if finality stalled for some reason and the last finalized block is more than 256 blocks
-behind, a pruned full node will not be able to sync to the network.
-
 Archive nodes are used by utilities that need past information - like block explorers, council
 scanners, discussion platforms like [Polkassembly](https://polkassembly.io), and others. They need
-to be able to look at past on-chain data. Full nodes are used by everyone else - they allow you to
-read the current state of the chain and to submit transactions directly to the chain without relying
-on a centralized infrastructure provider.
+to be able to look at past on-chain data.
+
+A **full node** is _pruned_, meaning it discards all information older than a specific number of
+blocks (256 blocks by default), and the genesis block. A node that is pruned this way requires
```

Comment:
Why do you mention the genesis block explicit?

Reply:
Was here in the wiki before, and my intent is to imply you need a chainspec that includes this block (or the information to produce it?), so all nodes retain this at least.

Reply:
```suggestion
number (256 blocks by default), and the genesis block. A node that is pruned this way requires
```

Reply:
But I read this as it also discards the genesis block :D But I'm no native speaker

Reply:
yap. That is a good point. Updated.